Sources/Steepshot/Steepshot.Core/Clients/BaseServerClient.cs

```diff
@@ -407,5 +408,105 @@ public async Task<OperationResult<SubscriptionsModel>> CheckSubscriptions(User u
             var endpoint = $"{BaseUrl}/{GatewayVersion.V1P1}/subscriptions/{user.Login}/{user.PushesPlayerId}";
             return await HttpClient.Get<SubscriptionsModel>(endpoint, token);
         }
+
+        public async Task<PromoteResponse> FindPromoteBot(PromoteRequest promoteModel)
```

Comment:
use Task<OperationResult<BidBot[]>> and return some object (exception or result)